api/server.py

```diff
@@ -195,7 +204,9 @@ def profile_dataset(
 
 
 @app.post("/profile_model/{model_id}")
-def profile_model(model_id: str, document_id: str, redis=Depends(get_redis)) -> ExtractionJob:
+def profile_model(
+    model_id: str, document_id: str, redis=Depends(get_redis)
+) -> ExtractionJob:
```

Comment:
Function definitions should either be on a single line or have each param on a different line. Can we expand this out?